### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by dvarchev**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/dvarchev/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
```
Comment (dvarchev) [inline ReadMe.md:2]: Wow! :+1:

Comment (dvarchev) [inline ReadMe.md:2]: !fix

**Comment by dvarchev** [comment]
!fixed

File: ReadMe.md (hunk 1 above)
Comment (dvarchev) [inline ReadMe.md:2]: !fixed

Comment (dvarchev) [inline ReadMe.md:2]: :+1:

**Pull request merged**